scripts/check-release.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { version: releaseVersion } = require('../lerna.json')
+const { findPackages, printLog, logAndExit, executeCommand } = require('./utils')
+
+async function main() {
+  await checkGitTag()
+  await checkPackages()
+
+  printLog('Release check done.')
+}
+
+async function checkGitTag() {
+  printLog('Checking git tag')
+  const headRef = await executeCommand('git rev-parse HEAD')
+  let tagRef
+  try {
+    tagRef = await executeCommand(`git rev-list -n 1 v${releaseVersion} --`)
+  } catch (error) {
+    throw new Error(`Failed to find git tag reference: ${error}`)
+  }
+  if (tagRef !== headRef) {
+    throw new Error('Git tag not on HEAD')
+  }
+}
+
+async function checkPackages() {
+  const packages = await findPackages()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏FMU we only want to check browser SDK packages. If so what about renaming the check function to make it clearer and do an early filter after the `findPackages` call.